### ISSUE TITLE
HTTPS Insecure Certificates

### ISSUE THREAD

**Issue by omarpta** [headzoo/surf#17]
Hi, it's just an idea...
I had to modify browser.go file (buildClient func) to work with insecure certificates urls (https):
```
import (
      "crypto/tls"
)
```

```
 func (bow *Browser) buildClient() *http.Client {
    tr := &http.Transport{
        TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
        }
    client := &http.Client{Transport: tr}
    client.Jar = bow.cookies
    client.CheckRedirect = bow.shouldRedirect
    return client
}
```

Would be nice if it were configurable...
:)

**Comment by headzoo** [comment]
Thanks for the ticket. This has been fixed in the master ( a917cf235 ). `Browser` now has a `SetTransport()` method so you can set your own.

**Comment by ashishthedev** [comment]
I was about to raise a ticket for this very issue and then discovered this thread. It would be nice if you can also update the documentation at http://surf.readthedocs.io/api/packages/browser/

Thanks.
